src/hyp3_isce2/insar_tops_burst.py

```diff
@@ -12,6 +12,9 @@
 from typing import Optional
 
 import isce
+# https://github.com/isce-framework/isce2/issues/258
+root_logger = logging.getLogger()
+root_logger.setLevel('WARNING')
```

Comment:
We added an import of `isce` to `hyp3_isce2.__init__` to ensure no matter where we are in this package, the ISCE2 stuff is set up correctly. Can you move this there?
https://github.com/ASFHyP3/hyp3-isce2/blob/develop/src/hyp3_isce2/__init__.py#L6-L13